src/views/Organizacoes/index.js

```diff
@@ -1,5 +1,24 @@
 import React from 'react'
 
+import { Organizations } from '../../components/Organizations'
+import { OrganizationsIcon } from '../../assets/icons/OrganizationIcon'
+
+import { Content, Title, Text } from '../../styles/base-styles.js'
+
 export const Orgs = () => {
-  return <div>Orgs</div>
+  return (
+    <Content>
+      <Title>
+        <OrganizationsIcon width={'2rem'} color={'#514E4E'} />
```

Comment:
Adicionar mudança de tema na cor do ícone
```suggestion
        <OrganizationsIcon width={'2rem'} color={theme['main-font']} />
```

É só tu usar um hook useTheme nesse componente que tu consegue acessar o tema